DSCResources/MSFT_xDatabase/MSFT_xDatabase.psm1

```diff
@@ -8,7 +8,7 @@ DacPacExtractionError=Extracting DacPac for Db failed, continuing with Dac Deplo
 '@
 }
 
-$SmoServerLocation = "${env:ProgramFiles(x86)}\Microsoft SQL Server\110\SDK\Assemblies\Microsoft.SqlServer.Smo.dll"
+Import-Module -DisableNameChecking $PSScriptRoot\..\xDatabase_Common
```

Comment:
Per previous comments,
Can you add a comment mentioning an issue that we should fix this?
File this issue if it doesn't exist?